.gitmodules

```diff
@@ -81,7 +81,8 @@
 [submodule "third_party/tensorpipe"]
     ignore = dirty
     path = third_party/tensorpipe
-    url = https://github.com/pytorch/tensorpipe.git
+    url = https://github.com/ROCm/tensorpipe.git
+    branch = tp_rocm_60
```

Comment:
afaik, the "branch" directive doesn't really have any impact, it's the submodule commit change and the submodule url here that matters